building.md

```diff
@@ -1,23 +1,21 @@
-\page building Building the Project
```

Comment:
these are ugly sure but they are used by doxygen. i've not found a way to have our cake and eat it too here... we either have doxygen doc look nice - or github.